Update vision to v0.10.0 (#119)
mapbox/vision-ios-examples: CHANGELOG.md

# Changelog

## 1.0-28

- `MapboxVision` v0.10.0

> - Added detection of ConstructionCones

Reviewer: ```suggestion
- Added detection of construction cones
```

Author: fixed

> - General bugfixes

Reviewer 2: Should we describe specific issues that were fixed?

Author: I think no, they are related to the life cycle of Vision classes, not user actions, so it would be difficult to explain them without naming specific classes

Reviewer 2: I'm ok to leave it as it is. Probably it would be more convenient to give more details anyway, but it's better to discuss it later, no blockers to merge now.

> - Improved performance, especially at night

Reviewer 3: ```suggestion
- Improved quality of detection/segmentation, especially at night
```

Author: fixed

> - Improved segmentation, now it more focused on road specific elements such as  Crosswalk, Hood, MarkupDashed, MarkupDouble, MarkupOther, MarkupSolid, Other, Road, RoadEdge, Sidewalk

Reviewer: ```suggestion
- Improved segmentation, now it's more focused on road specific elements. New segmentation model recognizes the following classes: Crosswalk, Hood, MarkupDashed, MarkupDouble, MarkupOther, MarkupSolid, Other, Road, RoadEdge, Sidewalk
```

Author: fixed
